ansible/roles/elastic_beats/vars/platform.yml

```diff
@@ -2,7 +2,7 @@
 
 filebeat_platform_inputs:
   - type: log
-    enabled: "{{ drive_host_info.containers | length > 0 }}"
+    enabled: "{{ tenderdash_host_info.containers | length > 0 }}"
```

Comment:
Not sure it's a good idea because tenderdash is running on seed nodes but without Drive